toolbox/utils.py

```diff
@@ -442,6 +465,259 @@ def as_command_choices(*args: t.Any, **kwargs: t.Any) -> t.Sequence[hikari.Comma
     return _list_to_command_choices(choices)
 
 
+class MarkdownFormat(IntFlag):
+    NONE = 0
+    STRIKETHROUGH = 1
+    ITALIC = 2
+    BOLD = 4
+    UNDERLINE = 8
+    CODE_BLOCK = 16
+    MULTI_CODE_BLOCK = 32
+    QUOTE = 64
+    MULTI_QUOTE = 128
+    SPOILER = 256
+
+    ALL = STRIKETHROUGH | ITALIC | BOLD | UNDERLINE | CODE_BLOCK | MULTI_CODE_BLOCK | QUOTE | MULTI_QUOTE | SPOILER
+
+
+def remove_markdown(content: str) -> str:
+    """Removes the markdown formatting from discord messages.
```

Comment:
```suggestion
    """Removes the markdown formatting from Discord messages.
```
This applied to all functions.